frontend/components/icons/expand.tsx

```diff
@@ -0,0 +1,14 @@
+export const ExpandIcon = () => {
+  return (
+    <svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
suggestion (non blocking): I tend to clean out the xmlns attribute. Also, not sure how the icon system is set up here, but I've found removing inline fill/stroke attributes setting all icons to use fill: currentColor a nice way to make the icons more usable